Test OK locally, this fixes the "Could not start auxiliary console process" error when starting a container from GNS3 Server.

Stack trace was:

```console
During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/nix/store/v5l1x2zawwdndnav9vydmzz3zyfyx9d5-gns3-server-2.2.46/lib/python3.11/site-packages/gns3server/web/route.py", line 200, in control_schema
    await func(request, response)
  File "/nix/store/v5l1x2zawwdndnav9vydmzz3zyfyx9d5-gns3-server-2.2.46/lib/python3.11/site-packages/gns3server/handlers/api/compute/docker_handler.py", line 330, in update
    await container.update()
  File "/nix/store/v5l1x2zawwdndnav9vydmzz3zyfyx9d5-gns3-server-2.2.46/lib/python3.11/site-packages/gns3server/compute/docker/docker_vm.py", line 459, in update
    await self.start()
  File "/nix/store/v5l1x2zawwdndnav9vydmzz3zyfyx9d5-gns3-server-2.2.46/lib/python3.11/site-packages/gns3server/compute/docker/docker_vm.py", line 512, in start
    await self._start_aux()
  File "/nix/store/v5l1x2zawwdndnav9vydmzz3zyfyx9d5-gns3-server-2.2.46/lib/python3.11/site-packages/gns3server/compute/docker/docker_vm.py", line 539, in _start_aux
    raise DockerError("Could not start auxiliary console process: {}".format(e))
gns3server.compute.docker.docker_error.DockerError: Could not start auxiliary console process: [Errno 2] No such file or directory: 'script'
```

--- a/pkgs/applications/networking/gns3/server.nix
+++ b/pkgs/applications/networking/gns3/server.nix
@@ -10,6 +10,7 @@
 , stdenv
 , nixosTests
 , testers
+, util-linux
 , gns3-server
 }:
 
@@ -54,6 +55,9 @@ python3.pkgs.buildPythonApplication {
   postInstall = lib.optionalString (!stdenv.hostPlatform.isWindows) ''
     rm $out/bin/gns3loopback
   '';
+
+  # util-linux (script program) is required for Docker support
+  makeWrapperArgs = [ "--suffix PATH : ${lib.makeBinPath [ util-linux ]}" ];
 
   doCheck = true;
 

--- a/pkgs/applications/networking/gns3/server.nix
+++ b/pkgs/applications/networking/gns3/server.nix
@@ -4,7 +4,7 @@
 }:
 
 { lib
-, python3
+, python3Packages
 , fetchFromGitHub
 , pkgsStatic
 , stdenv
@@ -14,7 +14,7 @@
 , gns3-server
 }:
 
-python3.pkgs.buildPythonApplication {
+python3Packages.buildPythonApplication {
   pname = "gns3-server";
   inherit version;
 
@@ -30,7 +30,7 @@ python3.pkgs.buildPythonApplication {
     cp ${pkgsStatic.busybox}/bin/busybox gns3server/compute/docker/resources/bin/busybox
   '';
 
-  propagatedBuildInputs = with python3.pkgs; [
+  propagatedBuildInputs = with python3Packages; [
     aiofiles
     aiohttp
     aiohttp-cors
@@ -67,7 +67,7 @@ python3.pkgs.buildPythonApplication {
     export HOME=$(mktemp -d)
   '';
 
-  checkInputs = with python3.pkgs; [
+  checkInputs = with python3Packages; [
     pytest-aiohttp
     pytest-rerunfailures
     (pytestCheckHook.override { pytest = pytest_7; })

